### DISCUSSION TITLE
Update Installation Documentation

### DISCUSSION THREAD

**Discussion by badewanne1234** [briis/hass-weatherflow2mqtt#145] [General]
I just installed this addon on my HassOS environment and thought everything else comes with it as well according to the installation documentation. It is described in one sentence that there is an MQTT server required, however when you read on, at least I was under the impression, that the MQTT server will be installed with this addon too.

Would be worthwhile to add the info that the MQTT server must be setup prior to installing this addon. I went with Mosquitto broker as the MQTT server, then restarted weatherflow2mqtt and it worked instantly.

Just a thought...

**Comment by briis** [comment]
Thanks for the suggestion, I have now added [this section](https://github.com/briis/hass-weatherflow2mqtt#prerequisites) to the README.

**Reply by badewanne1234** [reply]
Cool, thanks

**Reply by GlennGoddard** [reply]
@briis The ReadMe (Overview) needs updating on Docker Hub.

**Reply by briis** [reply]
Thanks. Will do that a bit later

**Reply by briis** [reply]
It is now updated. Thanks for reminding me Glenn.
